### DISCUSSION TITLE
Vanish + Squaremap

### DISCUSSION THREAD

**Discussion by rockit7** [Zrips/CMI#7039] [Feature Requests]
A pretty simple feature request. Could we possibly make it so /cmi vanish will make players vanish on Squaremap? Thank you in advance.

**Comment by mrfloris** [comment]
Isn't it the other way around, that squaremap has to support the plugins it tries to offer options for like 'hide vanished players', - they are the ones in my opinion that have to support the plugins they're trying to hide vanished from, such as cmi integration.

**Comment by rockit7** [comment]
They said speak to you. You say speak to them. Hopefully, together you can work something out. Since then, I have created my own solution.

**Reply by mrfloris** [reply]
> They said speak to you. You say speak to them. Hopefully, together you can work something out. Since then, I have created my own solution.

It's easy for a third party plugin to offer a feature like players on a map, and say 'dont show vanished', and then NOT support the dozen plugins out there that offer vanished mode. 

So in my personal opinion, I feel it's on them. They offer that feature. It's easy for them to say 'oh just let another plugin add us'. They want to hook into cmi to not show vanished, cmi already offers the vanished player option. 

I am pretty sure if that plugin has support for worldguard regions that They've added wg region support, and that it's not worldguard adding support for that plugin. but ala. nitpicking :)
 

But, I don't dev CMI, maybe zrips is perfectly fine adding it.
